ssas/service/admin/router.go

```diff
@@ -0,0 +1,16 @@
+package admin
+
+import (
+	"net/http"
+
+	"github.com/CMSgov/bcda-app/bcda/monitoring"
+	"github.com/go-chi/chi"
+)
+
+func NewRouter(middlewares ...func(http.Handler) http.Handler) http.Handler {
+	r := chi.NewRouter()
+	m := monitoring.GetMonitor()
+	r.Use(middlewares...)
+	r.Post(m.WrapHandler("/auth/system", createSystem))
```

Comment:
Is this really going to be mounted at `/auth/system`?  If I understand correctly, that route is intended for a different caller, and it might get confusing to duplicate routes.

Reply:
I based the path on the tech spec (under "Administrative Endpoints"). @DeirdreHolub , should this be somewhere else?

Reply:
The tech spec was written before we changed the strategy for how we were going to integrate this service and the bcda api.

Reply:
Is the strategy documented somewhere?